pkg/asset/agent/manifests/nmstateconfig.go

```diff
@@ -180,9 +180,7 @@ func (n *NMStateConfig) Load(f asset.FileFetcher) (bool, error) {
 		nmStateConfigList = append(nmStateConfigList, &nmStateConfig)
 	}
 
-	log := logrus.New()
-	log.Level = logrus.WarnLevel
```

Comment:
I'm remembering now the below call is to a library that logs a bunch of stuff at INFO level that we don't want to see: https://github.com/openshift/installer/commit/166434db40d0d399b336f1994d2677f7b59da8f8
We might need to save and restore the level.

Reply:
Yeah now we get:
```
INFO[0000] Start configuring static network for 5 hosts  pkg=manifests
INFO[0000] Adding NMConnection file <eno1.nmconnection>  pkg=manifests
INFO[0000] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
INFO[0001] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
INFO[0002] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
INFO[0002] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
```

Using this doesn't seem to work in hiding those Info messages though, I may need to revert it to use the prior way.
```
        level := logrus.GetLevel()
        logrus.SetLevel(logrus.WarnLevel)
        staticNetworkConfigGenerator := staticnetworkconfig.New(logrus.WithField("pkg", "manifests"), staticnetworkconfig.Config{MaxConcurrentGenerations: 2})
        logrus.SetLevel(level)
```

Reply:
Oh, the logging is done by the actual calls to the returned structure, not by `New()`. So we need to do `defer logrus.SetLevel(level)` instead.

Reply:
Yeah using defer works, thanks. Updated.